<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lesson1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="task3_groupBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Task3</string>
    </property>
    <widget class="QTextEdit" name="task3_textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="task3_plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>271</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="task3_add_pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="task3_replace_pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Replace</string>
     </property>
    </widget>
    <widget class="QPushButton" name="task3_insert_pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert html</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="task2_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>329</width>
      <height>203</height>
     </rect>
    </property>
    <property name="title">
     <string>Task2</string>
    </property>
    <widget class="QLineEdit" name="task2_a_lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="task2_b_lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="task2_gamma_lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="task2_c_lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="task2_solve_pushButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Solve</string>
     </property>
    </widget>
    <widget class="QPushButton" name="task2_clear_pushButton">
     <property name="geometry">
      <rect>
       <x>197</x>
       <y>170</y>
       <width>124</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="task2_angle_groupBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>52</y>
       <width>120</width>
       <height>74</height>
      </rect>
     </property>
     <property name="title">
      <string> Angle in:</string>
     </property>
     <widget class="QRadioButton" name="task2_deg_radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>96</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>deg</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="task2_rad_radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>96</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>rad</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="task2_a_label">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>53</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>a:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="task2_b_label">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>83</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>b:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="task2_gamma_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>113</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>gamma:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="task2_c_label">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>142</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>c:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="task1_label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>25</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Find the third side of the triangle</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="task1_groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>319</width>
      <height>204</height>
     </rect>
    </property>
    <property name="title">
     <string>Task1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="task1_a_label">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>53</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>a:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="task1_b_label">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>84</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>b:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="task1_c_label">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>113</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>c:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="task1_result_label">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>144</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>result:</string>
     </property>
    </widget>
    <widget class="QLabel" name="task1_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>25</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Solve a quadratic equation: ax^2 + bx + c = 0 </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="task1_a_lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="task1_b_lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="task1_c_lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="task1_result_lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="task1_solve_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Solve</string>
     </property>
    </widget>
    <widget class="QPushButton" name="task1_clear_pushButton">
     <property name="geometry">
      <rect>
       <x>187</x>
       <y>170</y>
       <width>124</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
